src/engine/renderer/tr_model_md3.cpp

```diff
@@ -218,8 +218,8 @@ bool R_LoadMD3( model_t *mod, int lod, const void *buffer, const char *modName )
 			v->xyz[ 1 ] = LittleShort( md3xyz->xyz[ 1 ] ) * MD3_XYZ_SCALE;
 			v->xyz[ 2 ] = LittleShort( md3xyz->xyz[ 2 ] ) * MD3_XYZ_SCALE;
 
-			lat = (latLng >> 8) * (2.0f * M_PI / 255.0f);
-			lng = (latLng & 255) * (2.0f * M_PI / 255.0f);
+			lat = (latLng >> 8) * (Math::mul2_pi_f / 255.0f);
```

Comment:
Would rather use `2.0f * Math::pi_f`. "mul2_pi" is incomprehensible

Reply:
Unfortunately I cannot do `Math::2pi_f`. 😭️🤣️

Reply:
I think the ones that are just a power of 2 times some other constant should be removed. Especially if they are not in any of the other math libraries.

Reply:
I'm fine with those names, `mul2_pi` just means `mul(2, pi)` so `2π`, like `div2_pi` means `div(2, pi)` so `2/π`, this is consistant. It matches prefixed writing and function calling (which uses a prefixed convention by the way).

Also, this constant is used very often, and some math libraries like GLM also provide dedicated symbols for it as well, named `glm::two_pi()`. That GLM naming looks interesting at first glance but it quickly goes out of hands, for example with `inv(mul(2, pi)` so `1/2π`, I named it `inv_mul2_pi` but GLM named `one_over_two_pi` …

I'm not surprised GLM doesn't define a constant for `π/360`, as what I named `divpi_360`, GLM would name it `pi_over_three_hundred_and_sixty`… 🤦‍♀️️

Reply:
> That [GLM naming looks](https://glm.g-truc.net/0.9.9/api/a00290.html) interesting at first glance but it quickly goes out of hands, for example with `inv(mul(2, pi)` so `1/2π`, I named it `inv_mul2_pi` but GLM named `one_over_two_pi`…

The GLM naming is better since it follows mathematical terminology, so I would actually be able to tell what it means without having to go read the source. 

> Actually GLM define some constants I haven't defined, like `4/π` or `3/2π`…

WTF